compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/util/RepoUtils.java

```diff
@@ -18,23 +18,16 @@
 package org.wso2.ballerinalang.util;
```

Comment:
Shall we remove this class as well? We can move the remaining utils to ProjectUtils and use that class.